src/vs/editor/common/config/editorOptions.ts

```diff
@@ -5975,6 +5980,9 @@ export const EditorOptions = {
 	inDiffEditor: register(new EditorBooleanOption(
 		EditorOption.inDiffEditor, 'inDiffEditor', false
 	)),
+	allowVariableLineHeights: register(new EditorBooleanOption(
```

Comment:
Could we also place this between `accessibilityPageSize` and `ariaLabel` to maintain that these are sorted?

Reply:
I think the IEditorOptions, the `accessibilityPageSize ` is currently lower placed than the `ariaLabel`, ie: the interface is currently not sorted correctly.